templates/taxi/car_list.html

```diff
@@ -6,12 +6,17 @@ <h1>Car list</h1>
     <ul>
       {% for car in car_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
+          <a href="{% url "taxi:car-detail" pk=car.id %}">{{ car.id }}</a>
           {{ car.model }} ({{ car.manufacturer.name }})
         </li>
       {% endfor %}
+    <form action="" method="post">
+      {% csrf_token %}
+      <a class="btn btn-success" href="{% url "taxi:car-create"%}">create a car</a>
+    </form>
     </ul>
   {% else %}
     <p>There are no cars in taxi</p>
```

Comment:
so if there won't existing cars, u won't be able to create a new one ?
can u place a create button outside if statement ?